content/blog/2023-06-25-intruducing-continuous-profiling-skywalking-with-ebpf/index.md

```diff
@@ -0,0 +1,235 @@
+
+```shell
+export SW_OAP_IMAGE=apache/skywalking-oap-server:9.5.0
+export SW_UI_IMAGE=apache/skywalking-ui:9.5.0
+export SW_ROVER_IMAGE=apache/skywalking-rover:0.5.0
+
+export FEATURE_FLAGS=mesh-with-agent,single-node,elasticsearch,rover
+make deploy.kubernetes
+```
+
+After deployment is complete, please run the following script to open SkyWalking UI: http://localhost:8080/.
+
+```shell
+kubectl port-forward svc/ui 8080:8080 --namespace default
+```
+
+## Create Continuous Profiling Policy
+
+Currently, we can select the specific service that we wish to monitoring by clicking the **Services** item in the **Service Mesh** panel.
+
+![Continuous Profiling](./continuous-profiling.png)
+
+By clicking on the edit button in the above figure, we can edit the continuous analysis configuration for this service.
+
+![Edit Continuous Profiling Policy](./edit-continuous-profiling-policy.png)
+
+The following configuration are support:
+
+1. **Target Type**: Specifies the type of profiling task to be triggered when the threshold determination is met.
+2. **Items**: For profiling task of the same target, one or more validation items can be specified. 
+As long as one validation item meets the threshold determination, the corresponding performance analysis task will be launched.
+   1. **Monitor Type**: Specifies the type of monitoring to be carried out for the target process.
+   2. **Threshold**: Depending on the type of monitoring, you need to fill in the corresponding threshold to complete the verification work. 
+   3. **Period**: Specifies the number of recent seconds of data you want to monitor. 
+   4. **Count**: Determines the total number of seconds triggered within the recent period. 
+   5. **URI Regex/List**: This is applicable to HTTP monitoring types, allowing URL filtering.
+
+## Done
+
+After clicking the save button, you can see the currently created monitoring rules, as shown in the figure below:
+
+![Continuous Profiling Monitoring Processes](./continuous-profiling-monitoring.png)
+
+The data can be divided into the following parts:
+
+1. **Rule list**: On the left, you can see the rule list you have created.
+2. **Monitoring Summary List**: Once a rule is selected, you can see which pods and processes would be monitored by this rule. 
+It also summarizes how many profiling tasks have been triggered in the **last 48 hours** by the current pod or process, as well as the last trigger time. 
+This list is also sorted in descending order by the number of triggers to facilitate your quick review.
+
+When you click on a specific process, you can see the data displayed as shown in the figure below:
+
+![Continuous Profiling Triggered Tasks](./continuous-profiling-tasks.png)
+
+The current figure contains the following data contents:
+
+1. **Task Timeline**: It shows when the current process has triggered performance analysis tasks in the **past 48 hours**. And when the mouse hovers over a task, it would also display detailed information: 
```

Comment:
```suggestion
1. **Task Timeline**: It list all profiling tasks in the **past 48 hours**. And when the mouse hovers over a task, it would also display detailed information: 
```